libraries/python/src/internal/mhsa_network.py

```
@@ -36,7 +36,7 @@ def forward(self, inputs: Tensor) -> tuple[Tensor, Tensor]:
 
         shape = (batch_size, sequence_length, self.heads_count, self.heads_dimension)
 
-        # shape: (batch, heads_count, sequence_length, head_dimension)
+        # shape: (batch, heads_count, sequence_length, head_dimension)  # noqa: ERA001
```

Ruff (0.12.2)

Unused `noqa` directive (non-enabled: `ERA001`)

Remove unused `noqa` directive

(RUF100)